frontend/src/opponent/ScoutOpponent.jsx

```diff
@@ -0,0 +1,21 @@
+import CancelIcon from "@mui/icons-material/Cancel";
+import { useState } from "react";
+import OpponentCard from "./OpponentCard";
+
+function ScoutOpponent({ display, setDisplay, opponents }) {
+  function closeScout() {
+    setDisplay(false);
+  }
+
+  return (
+    <div style={{ display: display ? "block" : "none" }}>
```

Comment:
This can be done using jsx rather than with css - so when you render `ScoutOpponent` in the parent component you can check display there instead

```
{display && (
  <ScoutOpponent .. />
)}
```

Reply:
That is very cool and clever, implemented now